File: 2 курс/4 семестр/Разработка мобильных приложений/Курсовая работа/HomeMediaLibrary/app/src/main/java/com/rarmash/homemedialibrary/Activities/MainActivity.java
package com.rarmash.homemedialibrary.Activities;

import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;

import androidx.activity.OnBackPressedCallback;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;
import androidx.preference.PreferenceManager;

import com.google.firebase.database.FirebaseDatabase;
import com.rarmash.homemedialibrary.Constant;
import com.rarmash.homemedialibrary.Fragments.InfoFragment;
import com.rarmash.homemedialibrary.Fragments.MediaFragment;
import com.rarmash.homemedialibrary.Fragments.ProfileFragment;
import com.rarmash.homemedialibrary.R;
import com.rarmash.homemedialibrary.databinding.ActivityMainBinding;


public class MainActivity extends AppCompatActivity {
    ActivityMainBinding binding;
    FirebaseDatabase db;
    SharedPreferences sharedPref;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        replaceFragment(new MediaFragment());

        db = FirebaseDatabase.getInstance();

        binding.bottomNavigatiomView.setOnItemSelectedListener(item -> {
            int itemId = item.getItemId();
            if (itemId == R.id.mediaList) {
                replaceFragment(new MediaFragment());
            } else if (itemId == R.id.profile) {
                replaceFragment(new ProfileFragment());
            } else if (itemId == R.id.info) {
                replaceFragment(new InfoFragment());
            }
            return true;
        });
        getOnBackPressedDispatcher().addCallback(this, new OnBackPressedCallback(true) {
            @Override
            public void handleOnBackPressed() {

            }
        });
    }

    private void replaceFragment(Fragment fragment) {
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        fragmentTransaction.replace(R.id.frame_layout, fragment);
        fragmentTransaction.commit();
    }
}

File: 3 курс/5 семестр/Разработка мобильных приложений на языке Котлин/RMP_Prac3/app/src/main/java/com/rarmash/prac3/ui/MainActivity.kt
package com.rarmash.prac3.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentTransaction
import com.rarmash.prac3.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        if (savedInstanceState == null) {
//            val fragment1: FirstFragment = FirstFragment()
//            val fragmentManager: FragmentManager = supportFragmentManager
//            val fragmentTransaction: FragmentTransaction = fragmentManager.beginTransaction()
//            fragmentTransaction.replace(R.id.fragment_container, fragment1)
//            fragmentTransaction.commit()
//        }
    }
}